capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizedPerson < baseDecorator
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure you correct this typing error it raises a **nameError** when trying to test your app. The imported class name should be `BaseDecorator` and not `baseDecorator`.